docs/install/local-docker.md

```diff
@@ -0,0 +1,83 @@
+# Install MLRun locally using Docker
+
+You can install and use MLRun and Nuclio locally on your computer, this will not include all the services and elastic 
+scaling capabilities which you can get with the Kubernetes based deployment but is much simpler to start with.
+
+```{admonition} Note
+- Using Docker is limited to local, Nuclio, and Serving runtimes and local pipelines.
+```
+
+The installation is done using [`docker compose`](https://docs.docker.com/compose/) and will deploy MLRun service, 
+MLRun UI, Nuclio serverless engine, and optionally Jupyter server. 
```

Comment:
Use [`docker compose`](https://docs.docker.com/compose/) to install MLRun. It deploys the MLRun service, 
MLRun UI, Nuclio serverless engine, and optionally the Jupyter server.